common/djangoapps/xblock_django/management/commands/pull_xblock_translations.py

```diff
@@ -32,10 +31,6 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        xblock_translations_root = xmodule_api.get_python_locale_root()
```

Comment:
Why remove this check?  If someone tries to run this command outside the makefile, it seems like it should still raise this error.

Reply:
@feanil would you mind checking my questions here about moving the code outside the `edx-platform`?

Reply:
@brian-smith-tcril I'd love your thoughts on this as well but I think it makes sense to have the core functionality in edx_django_utils so that we can have it in other IDAs.  Any reason not to just do that now?

Reply:
The only reason I can think of would be to prevent more OEP-58 scope creep. The main questions I have are:
1. How much does moving the functionality now increase scope?
2. Does not moving the functionality now make it harder to move it later?

If the answer to 1 is non trivial and the answer to 2 isn't "a lot" then I'd argue we should wait. Different answers to those questions could definitely tip the scales though.

Reply:
> If the answer to 1 is non trivial and the answer to 2 isn't "a lot" then I'd argue we should wait.

It's unfortunately, 

1) not trivial. I need to re-design the functionality to ensure proper separation of responsibility for the refactor to be useful.
2) Yes, nothing blocks moving this out anytime. The only downside, is that we'd have less fresh developer (me) memory in the future in case we decided to refactor.

Reply:
Given those answers, I think it makes sense to document the move as something we'd like to do in an issue (to hopefully aid in refreshing developer memory), and move forward with this PR as is.

@feanil thoughts?

Reply:
That makes sense to me as well.  Does the documenting need to happen in this PR or should I go ahead and review this as is?

Reply:
> Does the documenting need to happen in this PR ...?

No, It seems to me the best place to document is GitHub issues is within `credentials` or `course-discovery` and reference this pull request for useful funcitonality to pull from.

>  or should I go ahead and review this as is

Yes please.